### PR TITLE
S01_US9 : Users' navbars

### PR DESCRIPTION
**Pull request by Erdnaxela45**

### CODE REVIEW

**Review by maxime-guichard — DISMISSED** [review]

**Comment by endricklosange** [comment]
il faut mettre tes ficher de navbar dans un dossier includes :p

File: templates/_navbar.html.twig [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+                <li><a class="dropdown-item" href="#">Étudiant</a></li>
+                <li><a class="dropdown-item" href="#">Entreprise</a></li>
+            </ul>
+            </li>
+            <li class="nav-item">
+            <a class="nav-link ms-5 me-5" href="#">Connexion</a>
+            </li>
+            <li class="nav-item">
+            <a class="nav-link ms-5 me-5" href="#">Contact</a>
+            </li>
```
Comment (LucyMerlier) [inline templates/_navbar.html.twig:24]: indentation

File: templates/_navbar_company.html.twig [hunk 2]
```diff
@@ -0,0 +1,27 @@
+<nav class="navbar navbar-expand-sm bg-secondary navbar-dark">
+    <a class="navbar-brand" href="#">
+    <img src="logo.jpg" alt="logo" class"navbar-logo">
+  </a>
```
Comment (LucyMerlier) [inline templates/_navbar_company.html.twig:4]: indentation

File: templates/_navbar_company.html.twig [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+            </li>
+            <li class="nav-item">
+                <a class="nav-link ms-5 me-5" href="#">Se déconnecter</a>
+            </li>
+        </ul>
+    </div>
+    <button class="navbar-toggler hidden-md-up pull-left " type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </button>
+</nav>
```
Comment (LucyMerlier) [inline templates/_navbar_company.html.twig:27]: so this file has been created but it's not used anywhere?

File: templates/_navbar_student.html.twig [hunk 4]
```diff
@@ -0,0 +1,24 @@
+<nav class="navbar navbar-expand-sm bg-secondary navbar-dark">
+    <a class="navbar-brand" href="#">
+    <img src="logo.jpg" alt="logo" class"navbar-logo">
+  </a>
```
Comment (LucyMerlier) [inline templates/_navbar_student.html.twig:4]: indentation

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
see comments + there is an empty "navbar.scss" file

**Review by maxime-guichard — APPROVED** [review]

**Review by Wowlfy — DISMISSED** [review]

**Review by maxime-guichard — DISMISSED** [review]

**Review by LucyMerlier — CHANGES_REQUESTED** [review]
there are some useless files (_navbar_student.html.twig for example)

File: templates/_navbar.html.twig [hunk 5]
```diff
@@ -0,0 +1,30 @@
...
+                <a class="nav-link ms-5 me-5" href="#">Connexion</a>
+            </li>
+            <li class="nav-item">
+                <a class="nav-link ms-5 me-5" href="#">Contact</a>
+            </li>
+        </ul>
+    </div>
+    <button class="navbar-toggler hidden-md-up pull-left " type="button" data-bs-toggle="collapse" data-bs-target="#navbarNavAltMarkup" aria-controls="navbarNavAltMarkup" aria-expanded="false" aria-label="Toggle navigation">
+        <span class="navbar-toggler-icon"></span>
+    </butto
```
Comment (LucyMerlier) [inline templates/_navbar.html.twig:29]: truncated button tag

File: templates/_navbar.html.twig [hunk 6]
```diff
@@ -0,0 +1,30 @@
...
+
```
Comment (LucyMerlier) [inline templates/_navbar.html.twig:30]: end of navbar is missing

**Review by LucyMerlier — CHANGES_REQUESTED** [review]

**Review by Wowlfy — DISMISSED** [review]

**Review by maxime-guichard — DISMISSED** [review]

**Review by LucyMerlier — APPROVED** [review]

**Pull request merged**